testsuite/features/profiles/prv.suse.net/Docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM minima-mirror.prv.suse.net/toaster-sles12sp3-products
```

Comment:
... and those ones would be `internal_prv` instead of `prv.suse.net`

(or `external_prv`? not sure how this will be open to uyuni contributors)

Reply:
prv is still internal SUSE network. so it is internal_prv